src/utils/componentName.ts

```diff
@@ -38,7 +38,9 @@ export const getComponentName = (
   type: VNodeTypes
 ): string => {
   if (isObjectComponent(type)) {
-    return getComponentNameInSetup(instance, type) || type.name || ''
+    return (
+      getComponentNameInSetup(instance, type) || type.name || type.__name || ''
```

Comment:
A comment could be nice to explain why we need `__name`

Reply:
Is it OK?
![image](https://user-images.githubusercontent.com/82131679/192142529-4860ea8d-d5c9-4053-a6d5-da5f9d54887e.png)

Reply:
Perfect